updating/updating-restricted-network-cluster.adoc

```diff
@@ -38,11 +38,31 @@ include::modules/cli-installing-cli.adoc[leveloffset=+3]
 
 // this file doesn't exist, so I'm including the one that should pick up more changes from Clayton's PR - modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
 
-include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
+include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+3]
 
-include::modules/update-mirror-repository.adoc[leveloffset=+2]
+[id="update-mirror-repository_updating-restricted-network-cluster"]
+=== Mirroring the {product-title} image repository
 
-include::modules/machine-health-checks-pausing.adoc[leveloffset=+3]
+Before you update a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
```

Comment:
```suggestion
You must, in a restricted network, mirror the required container images into that environment before you update a cluster on infrastructure that you provisioned. You can also mirror images in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
```

Reply:
I _think_ another way to reword that first sentence might be: "You must mirror the required container images in a restricted network before you update a cluster on infrastructure that you provisioned." 

I was trying to reword to get to the action that a user must take earlier, which emphasizes that this is a procedure. I don't know if this works though.

Reply:
I'll play around to see if there's a way to reword this to get what you're going for. 

I don't think I'd want to split up "infrastructure that you provisioned" and "in a restricted network", because I think the first part only makes sense to include in this context when you specify that it's provisioned in a restricted network. Though maybe a clean looking reword won't be feasible while keeping this super clunky noun phrase the way it is. I'll test different options!